source/guides/stacks/hpcrs.rst

```diff
@@ -0,0 +1,228 @@
+
+   .. code-block:: bash
+
+      . /usr/local/etc/bash_completion.d/singularity
+
+
+Converting Docker images to Singularity Images
+**********************************************
+
+#. Download d2s
+
+   :command:`d2s` os an open source tool to convert Docker images to Singularity images. You can use the script in the location where it is downloaded, or install it using the included :file:`setup.py` file with the :command:`python setup.py install`
+
+
+   .. code-block:: bash
+
+      git clone https://github.com/intel/stacks.git
+      cd stacks/hpcrs/d2s
+
+#. List local Docker images
+
+   .. code-block:: bash
+
+      python d2s.py --list_docker_images
+
+   Your output can appear like this:
+
+   .. code-block:: console
+
+      ==============================
+      Docker images present locally
+      ==============================
+      ID         NAME
+      0: clearlinux/stacks-dlrs-mkl
+      1: clearlinux/stacks-dlrs_2-mkl
+      ==============================
+
+
+
+#. Convert to Singularity images
+
+   To convert the Docker images to Singularity images, use the :command:`d2s` script with the ID numbers of the images you wish to convert.
+   We strongly recommend using one of the :file:`clearlinux/stacks-dlrs-mkl` or :file:`sysstacks/stacks-dlrs-mkl` based images for this guide.  Other images may be incompatible with expected configuration or filesystem options.
+
+   .. code-block:: bash
+
+      python d2s.py --convert_docker_images <ID_1> <ID_2>
+
+#. Use the Singularity image
+
+   To use the container shell to run workloads, launch the image and you will be dropped into the shell. The Singularity image name will be the same as the name of the Docker image, with slashes converted to underscores.
+
+   .. code-block:: bash
+
+      singularity shell <singularity image>
+
+    Using the example output above, after conversion you could launch the clearlinux/stacks-dlrs-mkl Singularity image with
+    :command:`singularity shell clearlinux_stacks-dlrs-mkl`
+
+Execute a multi-node benchmark on an HPC cluster
+************************************************
+The following example was executed on an Intel(r) Xeon(r) Processor-based HPC infrastructure. The following steps may need to be adjusted for different environments. See this `Intel Whitepaper`_ for more information.
+
+Running a ResNet50 workload multi-node
+--------------------------------------
+
+#. Download the TensorFlow benchmark
+
+   .. code-block:: bash
+
+      git clone http://github.com/tensorflow/benchmarks -b cnn_tf_v1.13_compatible
+
+#. Copy the Singularity image and the benchmark files to the HPC cluster environment
+
+# Install OpenMPI* if needed
+
+   If the HPC host does not have OpenMPI installed, install a custom local version in the user's home directory. This version must be  the same as the version installed in the DLRS container.  Follow the steps for `building OpenMPI`_ from their documentation.
```

Comment:
Hey @bd-dean. A minor nit on indentation. 
Please left-align Ln 165 with Ln 163 above it, starting with "Install...". 
This fixes the unintended restart of number sequence below.

Reply:
one day I will get it right. Done with latest commit.